### PR TITLE
Add support for span logs

### PR DESCRIPTION
**Pull request by hanwavefront**

### CODE REVIEW

File: src/Wavefront.OpenTracing.SDK.CSharp/WavefrontTracer.cs [hunk 1]
```diff
@@ -28,14 +29,12 @@ public class WavefrontTracer : ITracer
...
         private static readonly string DerivedMetricPrefix = "tracing.derived";
+        private static readonly string WavefrontGeneratedComponent = "wavefront-generated";
         private static readonly string InvocationSuffix = ".invocation";
         private static readonly string ErrorSuffix = ".error";
         private static readonly string TotalTimeSuffix = ".total_time.millis";
         private static readonly string DurationSuffix = ".duration.micros";
         private static readonly string OperationNameTag = "operationName";
-
-        private static readonly List<string> HeartbeaterComponents =
-            new List<string> { "wavefront-generated", "opentracing", "csharp" };
```
Comment (sushantdewan123) [inline src/Wavefront.OpenTracing.SDK.CSharp/WavefrontTracer.cs]: why is this deleted?

Comment (hanwavefront) [reply src/Wavefront.OpenTracing.SDK.CSharp/WavefrontTracer.cs]: got deleted by mistake, was resolved.
